### PR TITLE
New package: LibNyanpasu.clash-nyanpasu version 1.5.1

### PR DESCRIPTION
**Pull request by keiko233**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/148633)

### CODE REVIEW

**Comment by keiko233** [comment]
> @keiko233 please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement
> ### Contribution License Agreement
> This Contribution License Agreement (**“Agreement”**) is agreed to by the party signing below (**“You”**), and conveys certain license rights to Microsoft Corporation and its affiliates (“Microsoft”) for Your contributions to Microsoft open source projects. This Agreement is effective as of the latest signature date below.
> 
> 1. **Definitions**.
>    **“Code”** means the computer software code, whether in human-readable or machine-executable form,
>    that is delivered by You to Microsoft under this Agreement.
>    **“Project”** means any of the projects owned or managed by Microsoft and offered under a license
>    approved by the Open Source Initiative ([www.opensource.org](http://www.opensource.org)).
>    **“Submit”** is the act of uploading, submitting, transmitting, or distributing code or other content to any
>    Project, including but not limited to communication on electronic mailing lists, source code control
>    systems, and issue tracking systems that are managed by, or on behalf of, the Project for the purpose of
>    discussing and improving that Project, but excluding communication that is conspicuously marked or
>    otherwise designated in writing by You as “Not a Submission.”
>    **“Submission”** means the Code and any other copyrightable material Submitted by You, including any
>    associated comments and documentation.
> 2. **Your Submission**. You must agree to the terms of this Agreement before making a Submission to any
>    Project. This Agreement covers any and all Submissions that You, now or in the future (except as
>    described in Section 4 below), Submit to any Project.
> 3. **Originality of Work**. You represent that each of Your Submissions is entirely Your original work.
>    Should You wish to Submit materials that are not Your original work, You may Submit them separately
>    to the Project if You (a) retain all copyright and license information that was in the materials as You
>    received them, (b) in the description accompanying Your Submission, include the phrase “Submission
>    containing materials of a third party:” followed by the names of the third party and any licenses or other
>    restrictions of which You are aware, and (c) follow any other instructions in the Project’s written
>    guidelines concerning Submissions.
> 4. **Your Employer**. References to “employer” in this Agreement include Your employer or anyone else
>    for whom You are acting in making Your Submission, e.g. as a contractor, vendor, or agent. If Your
>    Submission is made in the course of Your work for an employer or Your employer has intellectual
>    property rights in Your Submission by contract or applicable law, You must secure permission from Your
>    employer to make the Submission before signing this Agreement. In that case, the term “You” in this
>    Agreement will refer to You and the employer collectively. If You change employers in the future and
>    desire to Submit additional Submissions for the new employer, then You agree to sign a new Agreement
>    and secure permission from the new employer before Submitting those Submissions.
> 5. **Licenses**.
> 
> * **Copyright License**. You grant Microsoft, and those who receive the Submission directly or
>   indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license in the
>   Submission to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute
>   the Submission and such derivative works, and to sublicense any or all of the foregoing rights to third
>   parties.
> * **Patent License**. You grant Microsoft, and those who receive the Submission directly or
>   indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license under
>   Your patent claims that are necessarily infringed by the Submission or the combination of the
>   Submission with the Project to which it was Submitted to make, have made, use, offer to sell, sell and
>   import or otherwise dispose of the Submission alone or with the Project.
> * **Other Rights Reserved**. Each party reserves all rights not expressly granted in this Agreement.
>   No additional licenses or rights whatsoever (including, without limitation, any implied licenses) are
>   granted by implication, exhaustion, estoppel or otherwise.
> 
> 6. **Representations and Warranties**. You represent that You are legally entitled to grant the above
>    licenses. You represent that each of Your Submissions is entirely Your original work (except as You may
>    have disclosed under Section 3). You represent that You have secured permission from Your employer to
>    make the Submission in cases where Your Submission is made in the course of Your work for Your
>    employer or Your employer has intellectual property rights in Your Submission by contract or applicable
>    law. If You are signing this Agreement on behalf of Your employer, You represent and warrant that You
>    have the necessary authority to bind the listed employer to the obligations contained in this Agreement.
>    You are not expected to provide support for Your Submission, unless You choose to do so. UNLESS
>    REQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING, AND EXCEPT FOR THE WARRANTIES
>    EXPRESSLY STATED IN SECTIONS 3, 4, AND 6, THE SUBMISSION PROVIDED UNDER THIS AGREEMENT IS
>    PROVIDED WITHOUT WARRANTY OF ANY KIND, INCLUDING, BUT NOT LIMITED TO, ANY WARRANTY OF
>    NONINFRINGEMENT, MERCHANTABILITY, OR FITNESS FOR A PARTICULAR PURPOSE.
> 7. **Notice to Microsoft**. You agree to notify Microsoft in writing of any facts or circumstances of which
>    You later become aware that would make Your representations in this Agreement inaccurate in any
>    respect.
> 8. **Information about Submissions**. You agree that contributions to Projects and information about
>    contributions may be maintained indefinitely and disclosed publicly, including Your name and other
>    information that You submit with Your Submission.
> 9. **Governing Law/Jurisdiction**. This Agreement is governed by the laws of the State of Washington, and
>    the parties consent to exclusive jurisdiction and venue in the federal courts sitting in King County,
>    Washington, unless no federal subject matter jurisdiction exists, in which case the parties consent to
>    exclusive jurisdiction and venue in the Superior Court of King County, Washington. The parties waive all
>    defenses of lack of personal jurisdiction and forum non-conveniens.
> 10. **Entire Agreement/Assignment**. This Agreement is the entire agreement between the parties, and
>     supersedes any and all prior agreements, understandings or communications, written or oral, between
>     the parties relating to the subject matter hereof. This Agreement may be assigned by Microsoft.

@microsoft-github-policy-service agree [company="{your company}"]

**Comment by keiko233** [comment]
> @keiko233 the command you issued was incorrect. Please try again.
> 
> Examples are:
> 
> ```
> @microsoft-github-policy-service agree
> ```
> 
> and
> 
> ```
> @microsoft-github-policy-service agree company="your company"
> ```

@microsoft-github-policy-service agree company="Lib Nyanpasu"

**Comment by keiko233** [comment]
> @microsoft-github-policy-service agree

@microsoft-github-policy-service agree

**Comment by keiko233** [comment]
> @keiko233 the command you issued was incorrect. Please try again.
> 
> Examples are:
> 
> ```
> @microsoft-github-policy-service agree
> ```
> 
> and
> 
> ```
> @microsoft-github-policy-service agree company="your company"
> ```

@microsoft-github-policy-service agree

**Comment by keiko233** [comment]
@microsoft-github-policy-service agree

File: manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with komac v2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: moe.elaina.clash.nyanpasu
```
Comment (Trenly) [inline manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.installer.yaml:4]:
```suggestion
PackageIdentifier: LibNyanpasu.clash-nyanpasu
```
`Publisher.Package` format is preferred for GitHub releases

File: manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Created with komac v2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: moe.elaina.clash.nyanpasu
```
Comment (Trenly) [inline manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: LibNyanpasu.clash-nyanpasu
```

File: manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with komac v2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: moe.elaina.clash.nyanpasu
```
Comment (Trenly) [inline manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.yaml:4]:
```suggestion
PackageIdentifier: LibNyanpasu.clash-nyanpasu
```

File: manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Publisher: Lib Nyanpasu Org
+PublisherUrl: https://github.com/LibNyanpasu
+Author: Lin Nyanpasu
+PackageName: Clash Nyanpasu
+PackageUrl: https://nyanpasu.elaina.moe/
+License: GPL-3.0
+LicenseUrl: https://github.com/LibNyanpasu/clash-nyanpasu/blob/main/LICENSE
+Copyright: © 2024 Clash Nyanpasu All Rights Reserved
+CopyrightUrl: https://nyanpasu.elaina.moe/
+ShortDescription: Clash Nyanpasu～(∠・ω< )⌒☆​
```
Comment (Trenly) [inline manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.locale.en-US.yaml:16]: Please provide a better description of what this application is, does, or is used for

File: manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+Description: A Clash GUI based on tauri.
+Moniker: vscode
```
Comment (Trenly) [inline manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.locale.en-US.yaml:18]:
```suggestion
# Moniker:
```
The `vscode` moniker is already in use by Visual Studio Code and cannot be used here

File: manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# Created with komac v2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: moe.elaina.clash.nyanpasu
+PackageVersion: 1.5.1
+PackageLocale: en-US
+Publisher: Lib Nyanpasu Org
```
Comment (Trenly) [inline manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.locale.en-US.yaml:7]:
```suggestion
Publisher: elaina
```
This should match what is written to control panel; If it does not, `AppsAndFeaturesEntries` are required in the installer manifest

File: manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: moe.elaina.clash.nyanpasu
+PackageVersion: 1.5.1
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2024-04-08
```
Comment (Trenly) [inline manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.installer.yaml:11]:
```suggestion
ReleaseDate: 2024-04-08
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
The application will not launch if VCRedist is not installed

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Verify: 
image

File: manifests/l/LibNyanpasu/clash-nyanpasu/1.5.1/LibNyanpasu.clash-nyanpasu.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+ReleaseDate: 2024-04-08
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/LibNyanpasu/clash-nyanpasu/releases/download/v1.5.1/Clash.Nyanpasu_1.5.1_x64-setup.exe
+  InstallerSha256: 47AE24A9F099926B0364EA6D472CD7FEFBCB7F15FB044D72460A64FF7F478B3B
+ManifestType: installer
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/l/LibNyanpasu/clash-nyanpasu/1.5.1/LibNyanpasu.clash-nyanpasu.installer.yaml:20]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/l/LibNyanpasu/clash-nyanpasu/1.5.1/LibNyanpasu.clash-nyanpasu.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+- clash-nyanpasu
+- clash-rs
+- clash-verge
+- mihomo
+- rust
+- tauri
+- tauri-app
+ReleaseNotesUrl: https://github.com/LibNyanpasu/clash-nyanpasu/blob/main/CHANGELOG.md#151-2024-04-08
+ManifestType: defaultLocale
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/l/LibNyanpasu/clash-nyanpasu/1.5.1/LibNyanpasu.clash-nyanpasu.locale.en-US.yaml:31]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/l/LibNyanpasu/clash-nyanpasu/1.5.1/LibNyanpasu.clash-nyanpasu.yaml [hunk 10]
```diff
@@ -0,0 +1,8 @@
+# Created with komac v2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: LibNyanpasu.clash-nyanpasu
+PackageVersion: 1.5.1
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/l/LibNyanpasu/clash-nyanpasu/1.5.1/LibNyanpasu.clash-nyanpasu.yaml:8]:
```suggestion
ManifestVersion: 1.6.0
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @keiko233,

We've depreciated this manifest version, but most fields are identical to the newest version. Usually, just increasing the manifest version number is enough to pass this check.

File: manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.locale.en-US.yaml (hunk 6 above)
Comment (stephengillie) [reply manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.locale.en-US.yaml]: I'm not seeing any Publisher listed in the [Apps and features](https://github.com/microsoft/winget-pkgs/pull/148633#issuecomment-2052292193) page entry. This [manifest field](https://github.com/microsoft/winget-pkgs/blob/1d9936235b46cc3e4453c5f63e4489aace453890/manifests/l/LibNyanpasu/clash-nyanpasu/1.5.1/LibNyanpasu.clash-nyanpasu.locale.en-US.yaml#L7) was updated to be eponymous, is that accurate?

**Comment by keiko233** [comment]
> Hi @keiko233,
> 
> We've depreciated this manifest version, but most fields are identical to the newest version. Usually, just increasing the manifest version number is enough to pass this check.

fixed. thanks.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**